### PR TITLE
Allow a blank gender in User model validation

### PR DESCRIPTION
**Pull request by omgitsbillryan**

### CODE REVIEW

File: app/controllers/v0/sessions_controller.rb [hunk 1]
```diff
@@ -84,7 +84,7 @@ def saml_user
 
     def create_saml_user
       user = User.new(user_attributes)
-      user = Decorators::MviUserDecorator.new(user).create unless user.loa1?
+      user = Decorators::MviUserDecorator.new(user).create unless user.loa1? # || user.gender.nil?
```
Comment (omgitsbillryan) [inline app/controllers/v0/sessions_controller.rb]: i added this as a comment just to bring attention to it. 

@kreek does it make sense to try and decorate the user if gender is known to be `nil`?

Comment (kreek) [inline app/controllers/v0/sessions_controller.rb]: @omgitsbillryan did some more testing and sometimes MVI would find a user without middle name so yes I think I think we should still try, worst case they are not found and the user continues without a va_profile.

**Comment by kreek** [comment]
I think we should still hit MVI without middle name but everything else LGTM!

**Comment by omgitsbillryan** [comment]
@kreek i think i'll merge this PR - but your comment makes me think that we should perform loa3 user model validation, but never let the exception bubble up.  we should swallow it no matter what and just march on.  This goes for `gender` or any other attribute.  Does that sound right?

**Comment by omgitsbillryan** [comment]
I'm going to merge this for now, but I've logged a new issue that is a bit more of an all-encompassing solution.

**Pull request merged**